src/sentry/options/defaults.py

```diff
@@ -582,6 +582,10 @@
 # as the transition from UseCaseKey to UseCaseID occurs
 register("sentry-metrics.indexer.cache-key-rollout-rate", default=0.0)
 
+# A option for double writing old and new cache keys
+# for the same transition
+register("sentry-metrics.indexer.cache-key-double-write", default=False)
```

Comment:
Are we going to discard the option above (`sentry-metrics.indexer.cache-key-rollout-rate`)? Or are they used in conjunction

Reply:
We would end up discarding it. If we go the double-write path then we won't need it. I've just kept it in case we have any new concerns with the double-write approach.